tests/r/simple/runtime.txt

```diff
@@ -1 +1 @@
-r-2020-07-07
+r-2022-01-07
```

Comment:
Does this need to be bumped? Or could we add it as a new test case?

Reply:
I unbumped it!